spec/lib/terraforming/resource/iam_group_spec.rb

```diff
@@ -10,28 +10,40 @@ module Resource
       let(:groups) do
         [
           {
-            path: "/",
-            group_name: "hoge",
-            group_id: "ABCDEFGHIJKLMN1234567",
-            arn: "arn:aws:iam::123456789012:group/hoge",
-            create_date: Time.parse("2015-04-01 12:34:56 UTC"),
+            path: '/',
+            group_name: 'hoge',
+            group_id: 'ABCDEFGHIJKLMN1234567',
+            arn: 'arn:aws:iam::123456789012:group/hoge',
+            create_date: Time.parse('2015-04-01 12:34:56 UTC'),
```

Comment:
__[rubocop]__
* Avoid comma after the last item of a hash. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)